src/app/core/entity/edit-entity-type-dropdown/edit-entity-type-dropdown.component.ts

```diff
@@ -18,16 +11,7 @@ import { EntityRegistry } from "../database-entity.decorator";
 @Component({
   selector: "app-edit-entity-type-dropdown",
   templateUrl: "./edit-entity-type-dropdown.component.html",
-  imports: [
-    BasicAutocompleteComponent,
-    FaIconComponent,
-    MatFormField,
-    MatLabel,
-    MatTooltip,
-    NgIf,
-    ReactiveFormsModule,
-    MatInput,
-  ],
+  imports: [EntityTypeSelectComponent, MatFormField, MatLabel],
   standalone: true,
 })
 export class EditEntityTypeDropdownComponent
```

Comment:
Sorry, one more rename request, to also exactly align this with the other components' naming: 
EditEntityTypeDropdown --> EditEntityType (including test and folder, as usual)